diff -Naur -x '*~' -x '*.orig' hydra-7.1-src/hydra-gtk/src/interface.c hydra-7.1-src-fix-services-list/hydra-gtk/src/interface.c
--- hydra-7.1-src/hydra-gtk/src/interface.c	2011-10-02 10:06:15.000000000 +0200
+++ hydra-7.1-src-fix-services-list/hydra-gtk/src/interface.c	2011-10-31 18:02:08.833517218 +0100
@@ -230,8 +230,8 @@
   cmbProtocol_items = g_list_append(cmbProtocol_items, (gpointer) "http-proxy-urlenum");
   cmbProtocol_items = g_list_append(cmbProtocol_items, (gpointer) "https-head");
   cmbProtocol_items = g_list_append(cmbProtocol_items, (gpointer) "https-get");
-  cmbProtocol_items = g_list_append(cmbProtocol_items, (gpointer) "https-form-get");
-  cmbProtocol_items = g_list_append(cmbProtocol_items, (gpointer) "https-form-post");
+  cmbProtocol_items = g_list_append(cmbProtocol_items, (gpointer) "https-get-form");
+  cmbProtocol_items = g_list_append(cmbProtocol_items, (gpointer) "https-post-form");
   cmbProtocol_items = g_list_append(cmbProtocol_items, (gpointer) "icq");
   cmbProtocol_items = g_list_append(cmbProtocol_items, (gpointer) "irc");
   cmbProtocol_items = g_list_append(cmbProtocol_items, (gpointer) "imap");
